src/app/report/edit-display/edit-display.component.ts

```diff
@@ -244,8 +285,9 @@ export class EditDisplayComponent {
     }
   }
 
-  toggleToolTip(toolTip: MatTooltip): void {
-    toolTip.show();
-    setTimeout(() => toolTip.hide(), 2500);
+  setNewReport(message: string) {
+    this.editor.setNewReport(message);
```

Comment:
You have made a setNewReport function, but you don't use it? Or is this only for the encoding button?